### PR TITLE
3.5

### PR DESCRIPTION
**Pull request by thecutefinder**
## Description

Adds support for 3.5. Magix said to smileoflove on Discord that this can be done now.

Thanks to smileoflove for getting the protos and resources working. Protos and resources are from Yuuki.

I have tested this and can confirm it's working.

## Issues fixed by this PR
#2089
## Type of changes

- [ ] Bug fix
- [x] New feature 
- [x] Enhancement
- [ ] Documentation

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] My pull request is unique and no other pull requests have been opened for these changes
- [x] I have read the [Contributing note](https://github.com/Grasscutters/Grasscutter/blob/stable/CONTRIBUTING.md) and [Code of conduct](https://github.com/Grasscutters/Grasscutter/blob/development/CODE_OF_CONDUCT.md)
- [x] I am responsible for any copyright issues with my code if it occurs in the future.

### CODE REVIEW

**Comment by thecutefinder** [comment]
@KingRainbow44

**Comment by thecutefinder** [comment]
Fuck, I just realized, I need to add 3.5 resources as well. Give me a moment...

**Comment by thecutefinder** [comment]
Nevermind, it's fine.

File: build.gradle [hunk 1]
```diff
@@ -46,7 +46,7 @@ sourceCompatibility = JavaVersion.VERSION_17
 targetCompatibility = JavaVersion.VERSION_17
 
 group = 'xyz.grasscutters'
-version = '1.4.6'
+version = '1.5.1'
```
Comment (KingRainbow44) [inline build.gradle:49]: double bump?

**Comment by KingRainbow44** [comment]
video!video!video! besides that i might do a test, havent actually looked at the code yet bc github keeps killing my browser

**Comment by TalexDreamSoul** [comment]
这是来自QQ邮箱的假期自动回复邮件。你好，我最近正在休假中，无法亲自回复你的邮件。我将在假期结束后，尽快给你回复。

File: build.gradle (hunk 1 above)
Comment (thecutefinder) [reply build.gradle:49]: What version should I set it at?

**Comment by thecutefinder** [comment]
> video!video!video! besides that i might do a test, havent actually looked at the code yet bc github keeps killing my browser

I'll get the video in a moment here. I had tested it earlier today, but I'll do it again tomorrow. I'm tired right now.

File: build.gradle (hunk 1 above)
Comment (KingRainbow44) [reply build.gradle:49]: 1.4.7

**Comment by thecutefinder** [comment]
Working on getting a video right now

**Comment by thecutefinder** [comment]
The file is 769 MB and is too large to upload to GitHub. I'm putting it on Proton Drive and will upload it here once the upload finishes.

**Comment by thecutefinder** [comment]
https://drive.proton.me/urls/XWEDZX19J0#uL4eb2BoRKms

**Review by KingRainbow44 — APPROVED** [review]

**Pull request merged**